tests/py/dynamo/conversion/harness.py

```diff
@@ -353,6 +353,15 @@ def run_test_compare_tensor_attributes_only(
             debug=True,
         )
 
+        output_dtypes = None
+        if check_dtype:
```

Comment:
I believe your modification is correct, but I'm curious about the history. Why was it originally necessary to specify `output_dtypes` as an input in the `dynamic shape test` function? It seems that specifying `output_dtypes` is no longer needed, and your modification looks like the right approach.

Reply:
Thanks, I agree. it was included from other issue.